src/main/java/org/jabref/gui/frame/MainToolBar.java

```diff
@@ -118,12 +123,12 @@ private void createToolBar() {
                 globalSearchBar,
 
                 rightSpacer,
-
+            
```

Comment:
Do not add spaces. --> Place the cursor on a line with text in IntelliJ and then save.